inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndex.java

```diff
@@ -595,15 +596,18 @@ public void openPhysicalIndex()
         if (!isOpen) {
             // Only open if it is not already open
             try {
-                log.info("Opening index for project " + project.getName());
+                log.info("indexWriter was not open. Opening it for project " + project.getName());
```

Comment:
You can use

 `log.info("indexWriter was not open. Opening it for project [{}]", project.getName());`

like we do in other places.

Reply:
OK.